src/util/arith_tools.cpp

```diff
@@ -278,11 +278,23 @@ void mp_max(mp_integer &a, const mp_integer &b)
     a=b;
 }
 
+//#define B256
```

Comment:
This commit seems unrelated to the PR title and is adding in commented out code.  Is it worth at least adding one CI pass that enables this option?

Reply:
No, we should make a call on whether to use hex or the base 256.

Reply:
Agreed but that's not what this commit does at the moment.  Is there a case against storing binary numbers in binary internally?

Reply:
@martin-cs The only case I can think of is readability of .pretty() dumps. But that is hard to fix.